hack/observability/grafana/dashboards/controller-runtime.json

```diff
@@ -2681,7 +2826,7 @@
                 "uid": "PBFA97CFB590B2093"
               },
               "editorMode": "code",
-              "expr": "histogram_quantile(0.50, sum(rate(workqueue_queue_duration_seconds_bucket{pod=~\"$Pod\",name=~\"$Controller\"}[5m])) by (pod, name, le))",
```

Comment:
Changed the order of panels. Now the first one is work duration and the second one queue duration which seems more intuitive (I'm always looking for work duration)